apps/builder/app/shared/content-model.ts

```diff
@@ -424,7 +424,6 @@ export const richTextPlaceholders: Map<undefined | string, string> = new Map([
   ["blockquote", "Blockquote"],
   ["li", "List item"],
   ["a", "Link"],
-  ["span", "Span"],
```

Comment:
accidental?

Reply:
Nope, span is often used empty as non-textual container in radix components